packages/glibc227.rb

```diff
@@ -0,0 +1,134 @@
+require 'package'
+
+class Glibc227 < Package
+  description 'GNU C Library'
+  homepage 'https://www.gnu.org/software/libc/'
+  version '2.27'
+  source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.27.tar.xz'
+  source_sha256 '5172de54318ec0b7f2735e5a91d908afe1c9ca291fec16b5374d9faadfc1fc72'
+
+  depends_on 'make' => :build
```

Comment:
Not needed since make is already included in Chromebrew "core".

Reply:
Yes, but so is glibc. Glibc is installed before make is, so make would be a required package.

Reply:
Don't think of the install process.  That is irrelevant.  You will be compiling after crew is already installed anyway.  This package will eventually include pre-built binaries which will not require a compilation during install.